### ISSUE TITLE
sre_parse in _parse error: bad character range

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-gui#424]
https://app.getsentry.com/gns3/gns3-gui/group/68696274/

```
error: bad character range
(11 additional frame(s) were not displayed)
...
  File "c:\Python\64-bit\3.4\lib\re.py", line 288, in _compile
  File "c:\Python\64-bit\3.4\lib\sre_compile.py", line 465, in compile
  File "c:\Python\64-bit\3.4\lib\sre_parse.py", line 748, in parse
  File "c:\Python\64-bit\3.4\lib\sre_parse.py", line 358, in _parse_sub
  File "c:\Python\64-bit\3.4\lib\sre_parse.py", line 504, in _parse
```

**Comment by noplay** [comment]
The issue seem to be we pass to glob this:
Eigrp_and_HSRP_[Proof_On_Wrong-Design]_[Bad-Adjecency]

And glob think it's a regex.

**Comment by noplay** [comment]
glob escape is only available with Python 3.4
https://docs.python.org/3.4/library/glob.html

**Comment by noplay** [comment]
I have fixed for 1.3.5 and I will cleanup in 1.4 because we don't need the Python 3.3 patch

**Comment by noplay** [comment]
It's clean in 1.4. Python 3.4 only.
